app/retention-policy/server/startup/settings.js

```diff
@@ -28,18 +33,30 @@ settings.addGroup('RetentionPolicy', function() {
 		public: true,
 		i18nLabel: 'RetentionPolicy_Precision',
 		i18nDescription: 'RetentionPolicy_Precision_Description',
-		enableQuery: {
-			_id: 'RetentionPolicy_Enabled',
-			value: true,
-		},
+		enableQuery: [globalQuery, {
+			_id: 'RetentionPolicy_Advanced_Precision',
+			value: false,
+		}],
+	});
+
+	this.add('RetentionPolicy_Advanced_Precision', false, {
+		type: 'boolean',
+		public: true,
+		i18nLabel: 'RetentionPolicy_Advanced_Precision',
+		i18nDescription: 'RetentionPolicy_Advanced_Precision_Description',
+		enableQuery: globalQuery,
+	});
+
+	this.add('RetentionPolicy_Advanced_Precision_Cron', '*/30 * * * *', {
```

Comment:
since our cron parser is a little bit different (it has seconds as the first value), I think the most accurate default value would be:
```suggestion
	this.add('RetentionPolicy_Advanced_Precision_Cron', '0 */30 * * * *', {
```

Reply:
or maybe keep it simple and remove the seconds argument from https://github.com/RocketChat/Rocket.Chat/pull/18975/files#diff-69703d444418979235355dc9a536320aR72

this way everyone will be more familiar with the syntax.